src/Microsoft.Win32.Registry.AccessControl/src/Microsoft.Win32.Registry.AccessControl.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyName>Microsoft.Win32.Registry.AccessControl</AssemblyName>
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'netfx'">None</ResourcesSourceOutputDirectory>
-    <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsWindows)' != 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsWindows)' != 'true'">Access Control List (ACL) APIs are specific to the way in which Windows manages access to resources, and are not supported on this platform.</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
I agree it would be nice to have consistency, but I did this to avoid adding a resouce file to the project just for a single entry. Question is: Is it worth it?